addons/website/static/src/js/content/inject_dom.js

```diff
@@ -46,6 +46,12 @@ document.addEventListener('DOMContentLoaded', () => {
         htmlEl.dataset.country = country;
     }
 
+    const website_default_lang_name = session.website_default_lang_name;
+    if (website_default_lang_name) {
+        htmlEl.dataset.defaultLangName = website_default_lang_name;
+        htmlEl.dataset.langName = session.website_lang_name;
```

Comment:
I suggested `get_frontend_info` because I thought we could just do `session.xx` where we needed it (in the website_service), but since it's not possible (because it's the __session_info from the backend document) and you then had to "forward" / "pass" it from the iframe to the backend by setting it to the iframe html data attributes, it was better to just add it directly in those data attributes, which is done in the `website.layout` with `t-set="html_data"`.